com.github.kagami.boram.yml

```diff
@@ -0,0 +1,246 @@
+app-id: com.github.kagami.boram
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: Boram
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save files locally
+  - --filesystem=home
```

Comment:
it would be better to limit to `xdg-videos`, maybe also `xdg-download`

Reply:
I've picked `--filesystem=home` so the user wouldn't have issues with files in other directories inside home.

If possible, I think it might be nice to allow it.

Reply:
tbh I'm still worried about not allowing `/mnt`, since it's the default for auto-mounted partitions on Gnome Disks, so removing `$HOME` makes me even more worried.

Reply:
I've re-added `/mnt` to the manifest.

It's the default for auto-mounted partitions on Gnome Disks and I don't want to risk people thinking the app is broken because it can't access this directory.